docs/ai/toc.yml

```diff
@@ -5,6 +5,10 @@ items:
   items:
   - name: How generative AI and LLMs work
     href: conceptual/how-genai-and-llms-work.md
+  - name: Embeddings preserve semantic meaning
+    href: conceptual/embeddings.md
+  - name: Vector databases support semantic memory  
+    href: conceptual/vector-dbs.md
```

Comment:
Should these names match the doc names or is it okay to shorten them?

Reply:
done

Reply:
Genevieve suggested I should revert the file names